### DISCUSSION TITLE
Configurable PPEC Training Time

### DISCUSSION THREAD

**Discussion by AstroKatvis** [rmorgan001/GSServer#26] [Q&A] [answered]
Just starting out with GSS; great experience so far, thank you!

Generally it is advised on forums to run PPEC training for several work cycles. Does the GSS PPEC training offer any customized training cycles counts or time? It does not seem to be training for very long on my Skywatcher EQ8-R Pro mount, so I am somewhat dubious about the amount of training data it is processing. Is this a function of the mount or of GSS specifically? I believe in EQMod one can set the training time.
Thanks!

**Comment by rmorgan001** [comment, marked as answer]
GSS uses the SkyWatcher built in command set for PPEC training which only does one cycle.
